CHANGELOG.md

```diff
@@ -5,3 +5,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
 ## [Unreleased]
```

Comment:
Master branch pushes up to date images to docker hub every night.
Do you think that this changes should be under `Unreleased` section?
Maybe we should put a date instead, WDYT?

Reply:
The Keep a Changelog format would say they should be in `Unreleased` until a new release is cut. Once a release is cut and tagged, then you add a heading for that release to the changelog.

Reply:
I'm treating publishing in docker hub as "release" as far as we publish it with `0.11-latest` and `latest` tags, but may be I'm wrong.

Reply:
Just a date would not be appropriate for the changelog. Why is there no `0.11` in the changelog now?